TASK/TaskBoatMission.md

```diff
@@ -17,17 +19,33 @@ PED::SET_BLOCKING_OF_NON_TEMPORARY_EVENTS(pedDriver, 1);
 P8 appears to be driving style flag - see gtaforums.com/topic/822314-guide-driving-styles/ for documentation
 ```
 
-## Parameters
-* **pedDriver**: 
-* **boat**: 
-* **p2**: 
-* **p3**: 
-* **x**: 
-* **y**: 
-* **z**: 
-* **p7**: 
-* **maxSpeed**: 
-* **drivingStyle**: 
-* **p10**: 
-* **p11**: 
+```c
+enum BoatMissionFlags
+{
+  StopAtEnd = 1,
+  StopAtShore = 2,
+  AvoidShore = 4,
+  PreferForward = 8,
+  NeverStop = 16,
+  NeverNavMesh = 32,
+  NeverRoute = 64,
+  ForceBeached = 128,
+  UseWanderRoute = 256,
+  UseFleeRoute = 512,
+  NeverPause = 1024,
+}
+```
 
+## Parameters
+* **ped**: The ped to be tasked.
+* **boat**: The boats' entity handle.
+* **pedTarget**: The target ped (0 if not used)
```

Comment:
Wording should probably be changed here, maybe something like `(defaults to 0 if not set)` or `(default is 0)`